build-image/first-boot.bash

```diff
@@ -70,17 +70,23 @@ if is_raspbian || is_raspios; then
   rm -f "/etc/sudoers.d/010_pi-nopasswd"
 fi
 
-echo -n "$(timestamp) [openHABian] Changing default username and password... "
+echo -n "$(timestamp) [openHABian] Changing default username ... "
 # shellcheck disable=SC2154
 if [[ -v ${userName} ]] || ! id "$defaultUserAndGroup" &> /dev/null || id "$userName" &> /dev/null; then
```

Comment:
OK, so I understand the changes you are making but I think there might be an issue from older versions.

What does the `-v` flag do? I can't find it in the bash manual, I believe the `-v` instances should be replaced with `-n` right?

Reply:
right, changed that